xbmc/interfaces/legacy/ModuleXbmc.h

```diff
@@ -404,6 +404,14 @@ namespace XBMCAddon
     void audioResume();
 
     /**
+     * getUserAgent() -- Returns Kodis HTTP UserAgent string
```

Comment:
And also, if this gets rendered, an example user agent string for Jarvis would be extremely helpful